addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties

```diff
@@ -1,32 +1,278 @@
 # binding
-binding.max.name = MAX! Heizungssteuerung
-binding.max.description = Das Binding schafft eine Anbindung der MAX! Heizungssteuerung.
+binding.max.name = MAX! Heizungssteuerung Binding
+binding.max.description = Dieses Binding integriert die MAX! Heizungssteuerung (z.B. Heizk�rperthermostat, Heizk�rperthermostat+, Wandthermostat+, Eco Taster, Fensterkontakt).
+
+# bridge types
+thing-type.max.bridge.description = MAX! Cube LAN Gateway.
+
+# bridge types config groups
+thing-type.config.max.bridge.group.identification.label = Identifizierung
+thing-type.config.max.bridge.group.identification.description = Einstellungen f�r die Identifizierung des Ger�tes.
+
+thing-type.config.max.bridge.group.network.label = Netzwerk
+thing-type.config.max.bridge.group.network.description = Einstellungen f�r das Netzwerk.
+
+thing-type.config.max.bridge.group.device.label = Ger�t
+thing-type.config.max.bridge.group.device.description = Einstellungen f�r das Ger�t.
+
+thing-type.config.max.bridge.group.actions.label = Aktionen
+thing-type.config.max.bridge.group.actions.description = Einstellungen f�r Aktionen.
+
+# bridge types config
+thing-type.config.max.bridge.ipAddress.label = IP-Adresse
+thing-type.config.max.bridge.ipAddress.description = Lokale IP-Adresse oder Hostname des MAX! Cube LAN Gateway.
+
+thing-type.config.max.bridge.port.label = Port
+thing-type.config.max.bridge.port.description = Port des MAX! Cube LAN Gateway.
+
+thing-type.config.max.bridge.refreshInterval.label = Abfrageintervall
+thing-type.config.max.bridge.refreshInterval.description = Intervall zur Abfrage des MAX! Cube LAN Gateway (in Sekunden).
+
+thing-type.config.max.bridge.serialNumber.label = Seriennummer
+thing-type.config.max.bridge.serialNumber.description = Seriennummer des MAX! Cube LAN Gateway.
+
+thing-type.config.max.bridge.rfAddress.label = RF-Addresse
+thing-type.config.max.bridge.rfAddress.description = RF-Addresse des MAX! Cube LAN Gateway.
+
+thing-type.config.max.bridge.exclusive.label = Exklusivmodus
+thing-type.config.max.bridge.exclusive.description = Setzt den Exklusivmodus zur Nutzung des MAX! Cube LAN Gateways.
+
+thing-type.config.max.bridge.maxRequestsPerConnection.label = Verbindingsanzahl
+thing-type.config.max.bridge.maxRequestsPerConnection.description = Maximale Anzahl an Verbindungen zum MAX! Cube LAN Gateways.
+
+thing-type.config.max.bridge.ntpServer1.label = 1. NTP Zeitserver
+thing-type.config.max.bridge.ntpServer1.description = Erster NTP Zeitserver zur Abfrage des Datums und der Zeit.
```

Comment:
I would say "NTP-Server" instead of "NTP Zeitserver".
Since the "T" already stands for "Time"="Zeit".
Or Just "Zeitserver", but I would prefer "NTP-Server"

Reply:
@CWempe Thanks for review. "NTP-Server" sounds good.